package.json

```diff
@@ -20,9 +20,12 @@
   },
   "homepage": "https://github.com/MaxieMerge/bug-free-invention#readme",
   "dependencies": {
+    "bootstrap": "^5.0.2",
     "express": "^4.17.1",
     "express-session": "^1.17.2",
+    "i": "^0.3.6",
```

Comment:
I don't see this library being used anywhere. Can you double-check, and delete this if it's not needed?